sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/ConsoleResultPublisher.java

```diff
@@ -17,11 +17,22 @@
  */
 package org.apache.beam.sdk.loadtests;
 
-/** Writes {@link LoadTestResult} to console. */
-public class ConsoleResultPublisher {
+import static java.lang.String.format;
 
-  static void publish(LoadTestResult result) {
-    System.out.println(String.format("Total bytes: %s", result.getTotalBytesCount()));
-    System.out.println(String.format("Total time (sec): %s", result.getRuntime()));
+import java.util.List;
+import org.apache.beam.sdk.testutils.NamedTestResult;
+
+/** Writes load test metrics results to console. */
+class ConsoleResultPublisher {
+
+  static void publish(List<NamedTestResult> results, String testId, String timestamp) {
```

Comment:
I just wonder - isn't `testId` and `timestamp` already in each `NamedTestResult`?

Reply:
This is true. I figured it will be better to pass those as parameters rather than read from one of many records from the list.